source/Server.Contracts/ActionHandlers/Validation/ValidationExtensions.cs

```diff
@@ -33,7 +33,7 @@ public static class ValidationExtensions
 
                     return firstPackage != null && packages.Count() == 1 &&
                         !string.IsNullOrWhiteSpace(firstPackage.PackageId) &&
-                        !string.IsNullOrWhiteSpace(firstPackage.FeedId);
+                        !string.IsNullOrWhiteSpace(firstPackage.FeedIdOrName?.Value);
```

Comment:
An update to these validation extensions was necessary due to this new tinytype. Any Sashimi repo referencing this extension will need updated contracts.